server/call/operator.py

```diff
@@ -0,0 +1,63 @@
+"""Module which keeps track of all ongoing sessions and provides
+querying functionality to HTTP requesters."""
+import threading
+
+from daily import Daily
+from server.config import Config
+from server.call.session import Session
+
+
+class Operator():
+    _config: Config
+    _sessions: list[Session] = []
+    _is_shutting_down: bool
+
+    def __init__(self, config: Config):
+        self._config = config
+        self._is_shutting_down = False
+        Daily.init()
+
+    def create_session(self, room_duration_mins: int = None) -> str:
+        """Creates a session, which includes creating a Daily room."""
+        session = Session(self._config, self.clear_session, room_duration_mins)
+        self._sessions.append(session)
+        print("CREATED SESSION. ALL THREADS:", threading.active_count())
```

Comment:
This print (and the thread enumeration below) is temporary, just for the purposes of figuring out the ever-living dummy threads.